### DISCUSSION TITLE
API service hangs when specifying an external database.

### DISCUSSION THREAD

**Discussion by sphengle** [DependencyTrack/dependency-track#3189] [General]
So I got Dependency Track working just fine with the default YML file.  The only change I made was to the api port number.  I couldn't use 8081 as that was being used by something else.  I changed it to 8083.  All still OK.  Runs fine.

I then decided I wanted to use a SQL Server database.  I uncommented the database properties and set the connection string.  All good.  The database is populated with a schema and some of the tables have lookup data in them.  However the api service just hangs.  There is no response even when you directly put the URL into a browser.  Docker is not reporting anything unusual in the logs.

Anyone come across this issue before or have any idea what may be wrong?

**Edit**:  I found out what was wrong.  I didn't wait long enough.  While data is being synchronized into the datastore for the first time the web api is unavailable.  For me I had to wait quite a long time before it starting running.

**Comment by YaseenBabaShaik** [comment]
Hi How did you modify the port numbers?
Does DP track loads in browser after assigning new port in config files
